app/views/dashboard.html

```diff
@@ -195,7 +196,7 @@ <h4 translate="legend.ptratio">Pupil-Teacher Ratio</h4>
               </div>
             </div>
           </div>
-          <div class="container-fluid perfomance-gauges" ng-hide="selectedSchool|exists">
+          <div class="container-fluid perfomance-gauges" ng-hide="(selectedSchoolCode|exists) || (selectedPolyId|exists)">
```

Comment:
The logic in the html is getting kind of complex, but oh well it works and we're almost done, and I can't think of anything obviously better. :shipit: